Sources/Danger/BitBucketCloud.swift

```diff
@@ -228,6 +228,22 @@ extension BitBucketCloud {
 
         /// The user that created the comment
         public let user: User
+        
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 231;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)